packages/backend-defaults/src/entrypoints/auditor/auditorServiceFactory.ts

```diff
@@ -32,21 +35,40 @@ import { DefaultAuditorService } from './DefaultAuditorService';
 export const auditorServiceFactory = createServiceFactory({
   service: coreServices.auditor,
   deps: {
+    config: coreServices.rootConfig,
     logger: coreServices.logger,
     auth: coreServices.auth,
     httpAuth: coreServices.httpAuth,
     plugin: coreServices.pluginMetadata,
   },
-  factory({ logger, plugin, auth, httpAuth }) {
+  factory({ config, logger, plugin, auth, httpAuth }) {
     const auditLogger = logger.child({ isAuditEvent: true });
+    const auditorConfig = config.getOptionalConfig(CONFIG_ROOT_KEY);
+
+    const severityLogLevelMappings = {
+      low:
+        auditorConfig?.getOptionalString('severityLogLevelMappings.low') ??
+        'debug',
+      medium:
+        auditorConfig?.getOptionalString('severityLogLevelMappings.medium') ??
+        'info',
+      high:
+        auditorConfig?.getOptionalString('severityLogLevelMappings.high') ??
+        'info',
+      critical:
+        auditorConfig?.getOptionalString('severityLogLevelMappings.critical') ??
+        'info',
+    } as Record<
+      AuditorServiceEventSeverityLevel,
+      'debug' | 'info' | 'warn' | 'error'
```

Comment:
Hmm, in this case it might actually be good to verify that this is indeed what we're getting from config, since we'll be trying to call `undefined` otherwise

Reply:
I'm thinking about this case:

```yaml
backend:
  auditor:
    severityLogLevelMappings:
      low: definitelynotaloglevel
```